src/components/Post.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import User from './User';
+import CommentList from './CommentList';
+
+const Post = ({ postOne }) => (
+  <div className="post">
+    <p>
+      {'Post# - '}
+      {postOne.id}
+      {<br />}
+      {'Topic - '}
+      {postOne.title}
+    </p>
+    <p>{postOne.body}</p>
+    <User userOne={postOne.user} />
+    <CommentList commentOne={postOne.comments} />
```

Comment:
```suggestion
    <CommentList  comments={postOne.comments} />
```
Ты передаешь массив комментариев, а не один комментарий